base/docs/helpdb/Base.jl

```diff
@@ -8719,29 +8719,14 @@ alias the input `x` to modify it in-place.
 filt!
 
 """
-    ascii(::Array{UInt8,1})
+    ascii(s::AbstractString)
```

Comment:
You also should have run genstdlib and committed that at the same time, otherwise unrelated doc changes are ending up in multiple other PR's.